prep-html-script-tag/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Script Tag</title>
+    <body>
+      <script>
+        console.log('I have added JS to a script tag in the HTML!');
+      </script>
+      <script src="main.js"></script>
+      <script>
```

Comment:
Close! This 3rd script tag should not be here. Instead, in main.js you should have `console.log('I have added a script file via the src attribute!');` on the top line.